libcudacxx/include/cuda/std/__atomic/functions/cuda_local.h

```diff
@@ -40,12 +40,12 @@ _CCCL_DEVICE inline bool __cuda_is_local(const volatile void* __ptr)
 #  if defined(_LIBCUDACXX_ATOMIC_UNSAFE_AUTOMATIC_STORAGE)
   return false;
 // Only NVCC+NVRTC define __isLocal, so drop to PTX
-#  elif _CCCL_CUDACC_BELOW(12, 3) || _CCCL_CUDA_COMPILER(NVHPC)
+#  elif _CCCL_CUDACC_BELOW(12, 3) || _CCCL_CUDA_COMPILER(NVHPC) || defined(_LIBCUDACXX_FORCE_PTX_AUTOMATIC_STORAGE_PATH)
```

Comment:
Do we want to update the comments like `// ^^^ _CCCL_CUDACC_BELOW(12, 3) || _CCCL_CUDA_COMPILER(NVHPC) ^^^ / vvv other compiler vvv` to mention `defined(_LIBCUDACXX_FORCE_PTX_AUTOMATIC_STORAGE_PATH)` or is that not necessary because it's only used for forced testing of this code path?